doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1262,9 +1344,13 @@ A `subject` contains the following subfields:
 A `notification` object contains the following subfields:
 
 + `attributes`: List of attributes to be included in the notification message. If not specified (or empty), all attributes are included
-  in the notification
+  in the notification. It also defines the order in which attributes will appear in notifications when `attrsAs` "value" is used
+  (see "Notification messages" section).
 + `callback` : URL pointing to the service which will be invoked when a notification is generated. A NGSIv2
   compliant server must support `http` URL schema, other schemas (e.g. schemas for web sockets) could also be supported.
++ `headers`: a key-map of HTTP headers that will be included in the notification messages.
```

Comment:
what would happen if something like

{  
   'Content-Type': 'application/xml' 
}

is specified as header i.e a header which is inconsistent with headers that should be generated by default by the notification service